Matrix.py

```diff
@@ -0,0 +1,21 @@
+class Matrix:
+    def __init__(self, *args, **kwargs):
+        self.matrix_list = list(args)
```

Comment:
It is prefereble if the `__init__` method accepts only self and inint_data with anotation
Like a 
```suggestion
    def __init__(self: Self, init_data: list[list[int | float]]):
        self.matrix_list = init_data
```

Reply:
If you want, you can separate an annotation at the top of the file.
```python
Row = list[int | float]
MatrixType = list[Row]

class Matrix:
    def __init__(self: Self, init_data: MatrixType ):
        self.matrix_list = init_data
```

Reply:
If __init__ method takes only one argument, then it is not possible to implement the required functionality.
Poly-9 task suggests that the new Matrix object takes infinite number of arguments which are lists(rows) of the matrix

Reply:
I am eblan)0)0)))
You are right
I actualized task. It was
```python
x = Matrix([1, 1], [2, 2])
```

now
```python
x = Matrix([[1, 1], [2, 2]])
```

Reply:
Thank you for your attention